style.css

```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  box-sizing: border-box;
+  background-image: url("material1.png");
+  background-repeat: no-repeat;
+  background-size: 35%;
+  background-position: left top;
+}
+
+header {
+  display: flex;
+  margin: 2%;
+  justify-content: end;
+}
+
+.fa-bars {
+  position: inherit;
+  display: flex;
+  font-size: 2em;
+  justify-content: end;
+  margin: 15% 15px;
+  padding-top: 3%;
+}
+
+.logo {
+  font-family: "Playfair Display", serif;
+  font-size: 3.125rem;
+  color: #1a2236;
+  font-weight: normal;
+  letter-spacing: 2px;
+  line-height: 1em;
+  font-stretch: expanded;
+  padding: 15% 5% 0 5%;
+  background: transparent;
+  background-color: transparent;
+  margin: 5%;
+}
+
+.self {
+  font-family: "poppins", sans-serif;
+  font-size: 14px;
+  color: #1a2236;
+  font-weight: normal;
+  letter-spacing: 2px;
+  line-height: 1.5em;
+  font-stretch: expanded;
+  padding: 0 5% 0 5%;
+  margin: 5%;
+}
+
+.btn {
+  background: #1a2236;
+  color: #fff;
+  font-family: "poppins", sans-serif;
+  font-size: 1.2em;
+  font-weight: normal;
+  letter-spacing: 2px;
+  line-height: 1em;
```

Comment:
- [ ] Job working on the typo error made in the spelling of the word collaboration. However, you still need to add the suggestion given by the first reviewer that says that you need to implement a `hover` and `focus` interaction for the button as indicated in the project since your design template uses a button. check down here :point_down:
![Screenshot from 2021-11-25 23-15-18](https://user-images.githubusercontent.com/65068771/143500845-699d53a4-92cc-48bb-853c-a12ab8e08dec.png)
user something like this : 

| when you hover the button |
| ----------------------------------- | 
| .btn:hover { cursor: pointer} or follow this ![Screenshot from 2021-11-25 23-34-26](https://user-images.githubusercontent.com/65068771/143501712-35f897fb-ff0f-446b-be4f-efeb1fde18fe.png) |

| when you press the button |
| ----------------------------------- | 
| .btn:focus {  } and follow this ![Screenshot from 2021-11-25 23-34-39](https://user-images.githubusercontent.com/65068771/143501908-a8613e30-22dc-44fa-a26a-2718f0f89815.png) |

- [ ] I know you can do it :clap: :clap:  Kindly make this change and the next reviewer will approve it.

Reply:
I will do it!